kcp/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/kyma-project/kyma-watcher/kcp
+
+go 1.17
```

Comment:
Are we switching to go 1.18 for the project in a future PR?

Reply:
For kubebuilder init support I used this version, but I can use 1.18 since we do not need that anymore :)

Reply:
https://github.com/kyma-project/kyma-watcher/pull/19/commits/5756aa332816d177f6d97c79ad4d4b432c84e3bd